### PR TITLE
issue: 3865233: Fixing [UFM - TFS plugin] : Streaming into a Fluent server using the C CLX Forward library with IPv6 isn't working

### PR DESCRIPTION
**Pull request by ananalaghbar**
## What ?
The streaming in the TFS isn't working using the Fluent C Forward protocol with IPv6

## Why ?
Fixing [3865233](https://redmine.mellanox.com/issues/3865233)

## How ?
splitting the plugin_host_port string by ":" won't work with IPv6, since IPv6 includes (:) as part of their standard notation, removing the unnecessary string manipulation will fix the issue

### CODE REVIEW

**Review by vg12345 — APPROVED** [review]

File: plugins/fluentd_telemetry_plugin/src/fluentbit_writer.py [hunk 1]
```diff
@@ -181,9 +171,12 @@ def init_fb_writer(host, port, tag_prefix, timeout=120, use_c=True):
     if use_c:
         [lib, lib_path] = load_api_lib_from_path(LIB_RAW_MSGPACK_API_SO_PATH)
         ctx = {
-            'plugin_host_port': f'forward:{host}:{port}',
+            # 'plugin_host_port': f'forward:{host}:{port}',
             # 'plugin_host_port': 'stdout_raw:10.209.36.68:24226', # for testing
```
Comment (drorlevy) [inline plugins/fluentd_telemetry_plugin/src/fluentbit_writer.py:175]: remove IPs (public code)

File: plugins/fluentd_telemetry_plugin/src/fluentbit_writer.py [hunk 2]
```diff
@@ -202,6 +195,8 @@ def init_fb_writer(host, port, tag_prefix, timeout=120, use_c=True):
     # Example on how to set & use the FB writer
     _use_c = True
     _host = '10.209.36.67'
+    # _host = 'r-ufm254-hyp-03'
```
Comment (drorlevy) [inline plugins/fluentd_telemetry_plugin/src/fluentbit_writer.py:198]: remove addresses

File: plugins/fluentd_telemetry_plugin/src/fluentbit_writer.py [hunk 3]
```diff
@@ -80,19 +80,9 @@ def __init__(self, context):
...
-        if len(plugin_host_port_parts) != 3:
-            Logger.log_message("Failed to parse argument '%s'." % plugin_host_port, LOG_LEVELS.WARNING)
-            plugin_host_port = default_plugin_host_port
-            Logger.log_message("Fallback to default configuration '%s'" % plugin_host_port, LOG_LEVELS.WARNING)
-            plugin_host_port_parts = plugin_host_port.split(":")
-
-        self.plugin_name = plugin_host_port_parts[0]
-        self.host = plugin_host_port_parts[1]
-        self.port = plugin_host_port_parts[2]
+        self.plugin_name = context.get('plugin_name', 'forward')
```
[the remark this answers is not in the document]
Comment (ananalaghbar) [reply plugins/fluentd_telemetry_plugin/src/fluentbit_writer.py]: done

File: plugins/fluentd_telemetry_plugin/src/fluentbit_writer.py [hunk 4]
```diff
@@ -202,6 +195,8 @@ def init_fb_writer(host, port, tag_prefix, timeout=120, use_c=True):
     # Example on how to set & use the FB writer
     _use_c = True
     _host = '10.209.36.67'
+    # _host = 'r-ufm254-hyp-03'
+    # _host = 'fcfc:fcfc:209:36:219:b9ff:fe0a:ed14'
```
[the remark this answers is not in the document]
Comment (ananalaghbar) [reply plugins/fluentd_telemetry_plugin/src/fluentbit_writer.py]: done

File: plugins/fluentd_telemetry_plugin/src/fluentbit_writer.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ananalaghbar) [reply plugins/fluentd_telemetry_plugin/src/fluentbit_writer.py]: done

**Review by TalLerner — APPROVED** [review]

**Pull request merged**